.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ master ]
+   
```

Comment:
You need to add push to master event also to show codecov changes.
Refer https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/blob/master/.github/workflows/ci-build.yml#L4

Reply:
okay

Reply:
done